pkg/handler/testdata/cluster-scoped-resources/core/nodes.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
 items:
-- apiVersion: v1
```

Comment:
Please add a new fixture from a cilium sysdump for a cilium crd rather than re-using an existing one, because:
* This interferes with existing tests
* All the existing dumps have a listMeta of `apiVersion: v1`, `kind: List` because they ultimately come from `kubectl get -oyaml` which converts to a generic list. This means that the apiversion from the list is only correct by coincidence (because you changed a resource from the core group) and that  the `kind` from the list is not correct at all, which I suppose is why you left it in here

Reply:
For some reason, the Cilium sysdump has lists of nodes/pods/services/etc. that have the `apiVersion` only in the list, not in each element. I can add in `namespaces/kube-system` if you prefer, or would you rather I create another directory next to `testdata`?

Reply:
This format is returned by a standard k8s go client:
https://github.com/cilium/cilium-cli/blob/master/internal/k8s/client.go#L506

Reply:
Yeah, the k8s api only populates the typemeta on the list. Adding something in the kube-system namespace sounds good